core/plugins/org.fusesource.ide.foundation.core/src/org/fusesource/ide/foundation/core/util/Strings.java

```diff
@@ -32,6 +32,10 @@ public static boolean isEmpty(String text) {
         return text == null || text.length() == 0;
     }
     
+	public static boolean isNonEmptyAndNotOnlySpaces(Object value) {
```

Comment:
something is wrong with indention. 
method name looks weird. For a first step that is ok but we should consider refactoring the methods isEmpty and isBlank. We should have one method to do the null check and one method to do the trim().length>0 check imo. wdyt?

Reply:
also the two methods are checking quite different things in fact, one requires the parameter to be a String and one is calling toString() on the object